easybuild/framework/easyconfig/default.py

```diff
@@ -158,6 +158,11 @@
 
     # MODULES easyconfig parameters
     'whatis': [None, "List of brief (one line) package description entries", MODULES],
+    'usage': [None, "Usage instructions for the software", MODULES],
+    'docpaths': [None, "List of paths of installed documentation relative to package root dir", MODULES],
+    'docurls': [None, "List of URLs to package documentation", MODULES],
+    'support': [None, "Package-specific support/bug report address", MODULES],
+    'contact': [None, "Site contact for the package", MODULES],
```

Comment:
Well, the idea was to separate upstream contacts (`support`) from site-local contacts (`contact`).  The documentation of those two parameters already tries to make that clear.  But maybe the naming is sub-optimal.  I'm open for suggestions.